### DISCUSSION TITLE
Enable scrolling with keyboard.

### DISCUSSION THREAD

**Discussion by MP715** [immich-app/immich#7807] [Feature Request]
### The feature

Can't scroll up and down using arrow keys, Pgup, Pgdown, Ctrl+End, Ctrl+Home, etc.  I'm using Brave browser.  I didn't try Chrome, Firefox, etc.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by johanjongsma** [comment]
I would like keyboard shortcuts added for things like 'hide person', 'merge person' etc.

**Comment by ahpohl** [comment]
Controlling immich through keyboard shortcuts would be nice ('add photo to album' etc.). And scrolling the timeline with arrow keys and page up/down as suggested above.

**Comment by davidecavestro** [comment]
I confirm also Chrome desktop can't scroll using arrow/page keys. This forces users to click on the timeline, that is fine when you need to jump over the timeline, but cumbersome when you simply need to move back a day or week.

**Reply by alextran1502** [reply]
There is an extra step, you will have to focus the timeline i.e using the tab key to bring the focus to a photo on a timeline, or clicking on one photo and exit to create the focus. Then you can use the arrow key

**Reply by davidecavestro** [reply]
You are right, so there's an easy workaround; it works also selecting a photo (with a click) and deselecting it in order to get the focus and browse up/down using keys.
Indeed in the past I just tried clicking to check/unckeck the day checkbox, which doesn't lead to the same result.

**Comment by davidecavestro** [comment]
It works perfectly now without any extra step, on Chrome and immich v1.106.3
